src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -55,6 +55,7 @@
 #include <transformations/common_optimizations/transpose_reshape_elimination_for_matmul.hpp>
 #include "transformations/common_optimizations/matmul_multiply_fusion.hpp"
 #include "transformations/common_optimizations/mark_precision_sensitive_divides.hpp"
+#include "transformations/common_optimizations/resolve_gen_names_collisions.hpp"
```

Comment:
This transformation has nothing in common with common optimizations, so please move it to transformations folder

Reply:
Fixed